### PR TITLE
Alphabetize languages and services everywhere.

### PR DESCRIPTION
**Pull request by chadwcarlson**
Ordering for codetabs and sidebar alphabetized everywhere.

### CODE REVIEW

**Comment by Crell** [comment]
Alphabetizing the sidebar: :+1:.

Alphabetizing the tabs, I'm torn on.  On the one hand, it makes complete sense and is consistent.  On the other, PHP is still the vast majority of our customer base AFAIK, so making all of them pick another tab first seems like extra work for our most common users.

**Comment by chadwcarlson** [comment]
@Crell I figured this would be a push back point for PHP, and it makes sense. It made sense to me at the time to have the order consistent everywhere, but if we still want PHP to be first, how should codetabs be ordered?

Should we alphabetize the sidebar and then rank codetabs based on language popularity with customers throughout? Do we know how each of these rank?

File: src/SUMMARY.md [hunk 1]
```diff
@@ -58,17 +58,18 @@
 
 ## Languages
 
+
+* [Go](languages/go.md)
+* [Java](languages/java.md)
+* [Node.js](languages/nodejs.md)
+  * [Alternative Node.js install](languages/nodejs/nvm.md)
 * [PHP](languages/php.md)
   * [Extensions](languages/php/extensions.md)
```
Comment (otaviojava) [inline src/SUMMARY.md]: Nice!!!
I would like to do something similar to Java soon.

**Comment by Crell** [comment]
Rob has decreed(tm) that we go alphabetical, so alphabetical it is.

**Pull request merged**